### PR TITLE
Fix QA reported bugs

### PR DESCRIPTION
**Pull request by regiskuckaertz**
See https://docs.google.com/document/d/1DfX_8GA-_C5jzH2AdeGHQTpoO-K9lY47DmNZ4i9HZ2E/edit?ts=58106c9d

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
[CAPI multiple/single: fix browser testing bugs](https://trello.com/c/reMtvHgl/151-capi-multiple-single-fix-browser-testing-bugs)

File: src/_shared/js/ads.js [hunk 1]
```diff
@@ -26,5 +26,5 @@ export function addTrackingPixel(rootNode) {
 
 // Sets the 'href' of the glabs edition link to the correct URL.
 export function setEditionLink (edition, linkElement) {
-    linkElement.href = GLABS_EDITION[edition] || GLABS_EDITION.DEFAULT;
+    linkElement.href = '%%CLICK_URL_UNESC%%https://theguardian.com/' + (GLABS_EDITION[edition] || GLABS_EDITION.DEFAULT);
```
Comment (JonNorman) [inline src/_shared/js/ads.js]: Do we want to hardcode `theguardian` in here or use `host`?

File: src/_shared/js/capi-multiple.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ import { generatePicture } from './capi-images.js';
 import { setEditionLink } from './ads';
 import { URLSearchParams } from './utils';
 
+const clickMacro = '%%CLICK_URL_UNESC%%';
```
Comment (JonNorman) [inline src/_shared/js/capi-multiple.js]: Would it be worth adding this as a constant elsewhere where ''%%CLICK_URL_UNESC%%' is hardcoded?

Comment (regiskuckaertz) [inline src/_shared/js/capi-multiple.js]: Good call, yeah I will do that.

**Pull request merged**